playground/src/pages/CodingEnvironment.tsx

```diff
@@ -300,6 +349,13 @@ export default function CodingEnvironment(): JSX.Element {
           />
         </div>
       </div>
+      { proof ? 
+        <div className="flex">
+          <div className="absolute bottom-2 right-2 z-40">
+            <ActionButton label="Show Proof" onClick={showProof} />
```

Comment:
This button is hovering in a very odd position, it would make more sense to add it along side the `proof` and `verify` buttons instead, and make it clickable only after the proof is generated

Reply:
done, now there are many buttons at the top